chapter-03/recipe-05/fortran-example-3.5/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# set minimum cmake version
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+
+# project name and language
+project(recipe-05 LANGUAGES Fortran)
+
+find_package(OpenMP REQUIRED)
+
+# We do this in a pedestrian way because the Fortran support is relatively recent
+if(DEFINED CMAKE_Fortran_COMPILER_ID AND NOT DEFINED OpenMP_Fortran_FLAGS)
```

Comment:
OpenMP flags for Fortran are part of CMake 3.5, so this whole blurb can be (finally!) removed.

Reply:
Cool! I will nuke the blurb.